util/pastPerformance/testReleasesPerformance

```diff
@@ -47,6 +47,14 @@ fi
 export CHPL_HOME_ORIG=$CHPL_HOME
 export CHPL_TEST_UTIL_DIR=$CHPL_HOME_ORIG/util
 
+if [[ -z $CHPL_TEST_VENV_DIR && ! -z $CHPL_HOST_PLATFORM ]]; then
+    export CHPL_TEST_VENV_DIR=$CHPL_HOME/third-party/chpl-venv/install/$CHPL_HOST_PLATFORM/chpl-virtualenv
+fi
+if [[ ! -d $CHPL_TEST_VENV_DIR ]]; then
+    echo "ERROR: incorrect or missing CHPL_TEST_VENV_DIR or CHPL_HOST_PLATFORM"
```

Comment:
There are a couple of different things that can happen:
* user set CHPL_TEST_VENV_DIR incorrectly
* user set neither CHPL_TEST_VENV_DIR nor CHPL_HOST_PLATFORM
* the script set CHPL_TEST_VENV_DIR and it does not exist

Providing good diagnostics in all these cases will take quite a bit of code which IMO is not worth it.